cross-compiling/ignore_pkgs_scripts/foxy_ignore.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+if [ -z "$TARGET_ARCHITECTURE" ]; then
```

Comment:
Not related to this PR as it is everywhere, but this check is not required as we don't use this variable

Reply:
Deleted lines in https://github.com/irobot-ros/ros2-performance/pull/39/commits/19d484c35108bc6c1c57ae1f9ba5f6dab6a1dcdd